hil/ext/auth/keystone.py

```diff
@@ -5,14 +5,23 @@
 from keystonemiddleware.auth_token import filter_factory
 from flask import request
 from hil.flaskapp import app
-from hil.config import cfg
+from hil.config import cfg, core_schema
 from hil.model import Project
 from hil import auth, rest
 import logging
 import sys
 
 logger = rest.ContextLogger(logging.getLogger(__name__), {})
 
+core_schema[__name__] = {
+    'auth_url': str,
```

Comment:
Could probably check for a valid URL here.

Would be worth tightening the other fields here as well, where possible.

Reply:
I believe the only field I can check here is the `auth_url`.  I considered `auth_protocol` at least, but the authentication plugins available can differ depending on what's installed.